site/docs/5_references/1_nifi_cluster/1_nifi_cluster.md

```diff
@@ -88,12 +88,13 @@ spec:
 | rootProcessGroupId | string                                                              | contains the uuid of the root process group for this cluster.                               | if external type | -          |
 | secretRef          | \[ \][SecretReference](../4_nifi_parameter_context#secretreference) | reference the secret containing the informations required to authentiticate to the cluster. | if external type | -          |
 | proxyUrl           | string                                                              | defines the proxy required to query the NiFi cluster.                                       | if external type | -          |
-
```

Comment:
I also fixed a basic bug in the docs here, introduced by an extraneous newline, that resulted in a broken markdown table.

Reply:
This was also fixed in #162, so i've rebased on that.